backend/apps/readings/analysis.py

```diff
@@ -370,3 +370,114 @@ def compute_median_view_time(self):
             list_of_times.sort()
             median_view_time = statistics.median(list_of_times)
         return round(median_view_time)
+
+    def compute_mean_response_length(self):
+        """
+        Given a list of student response dicts,
+        return the mean character length (across all users) of the response
+        :return: float, mean number of characters in the user's response
+        """
+        list_of_responses = [r.response for r in self.responses]
+        mean_response_length = 0
+        for response in list_of_responses:
+            mean_response_length += len(response)
+        return round(mean_response_length / len(list_of_responses), 2)
+
+    def get_number_of_unique_students(self):
+        """
+        Count the number of unique students that gave responses
+        :return: the number of unique students that gave responses
+        """
+        unique_students = set()
+        for row in self.responses:
+            unique_students.add(row.student)
+        return len(unique_students)
+
+    @staticmethod
+    def description_has_relevant_words(story_meaning_description, relevant_words):
+        """
+        Determine if the user's description contains a word relevant to the story's meaning
+        :param story_meaning_description: The user's three word description of the story
+        :param relevant_words: a list of words which show an understanding of the story's meaning
+        :return True if the description contains one of the relevant words or relevant_words is
+        empty. False otherwise
+        """
+        if not relevant_words:
+            return True
+
+        lowercase_relevant_words = list(map(lambda s: s.lower(), relevant_words))
+        words_used_in_description = story_meaning_description.lower().split(' ')
+
+        for word in lowercase_relevant_words:
+            if word.lower() in words_used_in_description:
+                return True
+        return False
+
+    @staticmethod
```

Comment:
Since we're hard coding for now, I'd just get rid of this outside data file and wham a list into the one place where this is called. We're going to want to replace that with something better ASAP anyway (probably by taking instructor input from the frontend)